### PR TITLE
Added an unordered list of my favorite coffees

### PR DESCRIPTION
**Pull request by DerekMalone**
Altered index.html and added an unordered list of 3 items.

### CODE REVIEW

**Comment by tiffanyautumn** [comment]
👍

**Review by tiffanyautumn — APPROVED** [review]
👍

File: index.html [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Best Coffee Ever!</h1>
+    <h3>Heres a list of some of the best coffee ever</h3>
+    <ul>
+      <li>Lava Java - Peaberry</li>
+      <li>8th & Roash - Jetsetter</li>
```
Comment (Valerie-Freeman) [inline index.html]: Please fix this spelling error

**Review by Valerie-Freeman — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (DerekMalone) [reply index.html]: Please review changes once more, I fixed it.

**Review by Valerie-Freeman — APPROVED** [review]
👍

**Pull request merged**